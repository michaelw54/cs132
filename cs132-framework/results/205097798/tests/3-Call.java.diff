1,4c1,2
< ParseException: Encountered " <IDENTIFIER> "w_2 "" at line 2, column 1.
< Was expecting:
<     "return" ...
<     
---
> 42
> 99
