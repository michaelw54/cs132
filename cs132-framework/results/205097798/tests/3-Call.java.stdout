func Main()
    w4 = 4
    w2 = alloc(w4)
    if0 w2 goto null_2
    goto passNull_2
  null_2:
    error("null pointer")
  passNull_2:
    w4 = 4
    w3 = alloc(w4)
    if0 w3 goto null_3
    goto passNull_3
  null_3:
    error("null pointer")
  passNull_3:
    [w2 + 0] = w3
    w4 = @A_run
    [w3 + 0] = w4
    w1 = [w2 + 0]
    w1 = [w1 + 0]
    w0 = call w1(w2)
    print(w0)
    ret_ = 0
    return ret_
    
func A_run(this)
    w6 = 42
    print(w6)
    w7 = 99
    return w7
    
